sdk/servicebus/Azure.Messaging.ServiceBus/tests/Infrastructure/ServiceBusScope.cs

```diff
@@ -84,20 +58,29 @@ public static class ServiceBusScope
 
             caller = (caller.Length < 16) ? caller : caller.Substring(0, 15);
 
-            var azureSubscription = ServiceBusTestEnvironment.Instance.SubscriptionId;
-            var resourceGroup = ServiceBusTestEnvironment.Instance.ResourceGroup;
             var serviceBusNamespace = overrideNamespace ?? ServiceBusTestEnvironment.Instance.ServiceBusNamespace;
-            var token = await AcquireManagementTokenAsync().ConfigureAwait(false);
 
-            string CreateName() => $"{ Guid.NewGuid().ToString("D").Substring(0, 13) }-{ caller }";
+            var queueName = $"{ Guid.NewGuid().ToString("D").Substring(0, 13) }-{ caller }";
```

Comment:
This intentionally was a function so that each retry would attempt a different name, in the (very) rare case that we had a collision or Service Bus rejected the name for some other reason.  

To be honest, I don't remember what prompted us to do that, as it seems very unlikely.  Since it doesn't map well to the admin client's implicit retries, I'd agree it's not worth replicating unless we see weird issues.

Reply:
Interesting, I don't think we do this in any of our admin client tests.